doc/tags.adoc

```diff
@@ -0,0 +1,50 @@
+= Understanding The Tags
+
+We use tags to catetorize different types of test scenarios. With a combination of tags, we are able to execute on a specific ci profile.
+
+== List of Tags
+
+[cols=2]
+|===
+|Tag Name
+|Usage Scenario
+
+|`@admin`
+|Test scenarios that perform operations requiring the `cluster-admin` priviledge
+
+|`@destructive`
+|Test scenarios that perform operations which would result in failures of other parallel tests or create an outage/downtime to the cluster. Scenarios including: node draining, node restart, scaling down, etc.
+
+|`@inactive`
+|Inactive test scenarios, either outdated or no longer applicable. Skip tests with an `@inactive` tag
+
+|`@flaky`
+|Unreliable test that keeps failing and is recognized as test automation bug. Skip tests with a `@flaky` tag
+
+|`@prow_unstable`
```

Comment:
perhaps `prow_flaky` to be consistent ?

Reply:
Initially, I only wanted the `@flaky` tag. I realize we should give the team more time to triage their unstable tests in Prow, so used a different name to distinguish them.